CMakeLists.txt

```diff
@@ -72,8 +72,6 @@ set(run_e2e_tests ${original_run_e2e_tests})
 set(run_int_tests ${original_run_int_tests})
 set(run_unittests ${original_run_unittests})
 
-include_directories(${MACRO_UTILS_INC_FOLDER})
```

Comment:
This change seems fine to me (adding macro_utils as target_link_library to trigger adding the include).

For the other change (the package location), I'd like Ewerton to chime in. Added him to the review and pinged him.

Reply:
Actually I just ran the gated build on this PR to validate that all configs that IoT SDK builds would pass and the build failed.

That is because the IoT SDK team still supports some older versions of cmake which will not handle the interface target.

So having the include_directories seems to still be needed.
Is there any specific reason why it would not work for your case with the include_directories?

-- This version of CMake does not support interface targets. Only the Azure Macro Utils header file will be installed. To use Azure Macro Utils, simply add "MACRO_UTILS_INC_FOLDER" to your include directories as specified in the instructions on the GitHub README.
-- This version of CMake does not support interface targets. Only the Azure Macro Utils header file will be installed. To use Azure Macro Utils, simply add "MACRO_UTILS_INC_FOLDER" to your include directories as specified in the instructions on the GitHub README.
-- This version of CMake does not support interface targets. Only the Azure Macro Utils header file will be installed. To use Azure Macro Utils, simply add "MACRO_UTILS_INC_FOLDER" to your include directories as specified in the instructions on the GitHub README.


---
In reply to: [314459777](https://github.com/Azure/umock-c/pull/114#discussion_r314459777) [](ancestors = 314459777)